packages/rum-core/src/common/config-service.js

```diff
@@ -77,28 +77,19 @@ class Config {
       logLevel: 'warn',
       breakdownMetrics: false,
       ignoreTransactions: [],
-      errorThrottleLimit: 20,
-      errorThrottleInterval: 30000,
-      transactionThrottleLimit: 20,
-      transactionThrottleInterval: 30000,
-
+      eventsPerMin: 80,
```

Comment:
Another option for this might be just `eventsLimit`, not having the interval (minute) in the name helps if we want to change the interval in the future without having to changing the name. Another benefit is that we don't have to use shortened `min`  in the name.

Reply:
Didn't we think about this first and decide then its not super informative for the user and decided to use `eventsPerMin`? . I like `eventsLimit` as well.

Reply:
yeah, last we discussed we said to keep the interval in the name, but thinking about it again, it seems to me it's better to document the interval instead just keep the name simple.

Reply:
I can change it to `eventsLimit`. I like both so lets go for it.